### PR TITLE
CI changes for next_rails and removal of Webui::FlashHelper

### PR DESCRIPTION
**Pull request by dmarcoux**
There are still more spec failures in the `next_rails` CI check, those will be addressed in other PRs.

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Update Rails from 6.0.x to 6.1.x](https://trello.com/c/5oZbcuNM/1689-update-rails-from-60x-to-61x)

File: .circleci/config.yml [hunk 1]
```diff
@@ -51,6 +51,20 @@ aliases:
       path: ./src/api/log/
       destination: minitest
 
+  - &store_rspec_artifacts
```
Comment (hennevogel) [inline .circleci/config.yml:54]: Would you mind adding a step to compress logs? This is annoying me for some time already...

**Review by hennevogel — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (hennevogel) [reply .circleci/config.yml:54]:
```
➜  Downloads ls -lah test.log                                         
Permissions Size User   Date Modified Name
.rw-r--r--  191M hvogel 22 Jun 15:41  test.log
➜ Downloads time xz -T 0 test.log
xz -T 0 test.log  41,66s user 0,45s system 711% cpu 5,917 total
➜  Downloads ls -lah test.log.xz 
Permissions Size User   Date Modified Name
.rw-r--r--  6,9M hvogel 22 Jun 15:41  test.log.xz
```

Comment (dmarcoux) [reply .circleci/config.yml:54]: I would prefer to create a GitHub issue for that and tackle this later.

**Pull request merged**